<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsSound</class>
 <widget class="QWidget" name="SettingsSound">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>MIDI In Device:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sound card #1:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonConfigureSoundCard1">
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Sound card #2:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButtonConfigureSoundCard2">
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Sound card #3:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButtonConfigureSoundCard3">
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>

   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Sound card #4:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButtonConfigureSoundCard4">
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>


   <item row="5" column="1">
    <widget class="QComboBox" name="comboBoxMidiIn">
     <property name="maxVisibleItems">
      <number>30</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>MIDI Out Device:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="checkBoxMPU401">
     <property name="text">
      <string>Standalone MPU-401</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pushButtonConfigureMPU401">
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButtonConfigureMidiIn">
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBoxMidiOut">
     <property name="maxVisibleItems">
      <number>30</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButtonConfigureMidiOut">
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="checkBoxFloat32">
     <property name="text">
      <string>Use FLOAT32 sound</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>FM synth driver</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_1">
      <item>
       <widget class="QRadioButton" name="radioButtonNuked">
        <property name="text">
         <string>Nuked (more accurate)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonYMFM">
        <property name="text">
         <string>YMFM (faster)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxSoundCard1">
     <property name="maxVisibleItems">
      <number>30</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxSoundCard2">
     <property name="maxVisibleItems">
      <number>30</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBoxSoundCard3">
     <property name="maxVisibleItems">
      <number>30</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBoxSoundCard4">
     <property name="maxVisibleItems">
      <number>30</number>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
